fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+    /* your code here */
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(response => response.json())
+      .then(
+        result => {
+          this.setState({
+            users: result,
+            isLoading: false
+          });
+        },
+        error => {
+          this.setState({
+            isLoading: false,
+            error
+          });
+        }
+      );
```

Comment:
While this works this is the old style of handling promises, it is best to use `.then` for success and `.catch` for errors:
```JSX
.then(result => {
  this.setState({
    users: result,
    isLoading: false
  });
})
.catch(error => {
  // Do error handling here
  console.error(error);
});
```